src/index.ts

```diff
@@ -8,10 +8,22 @@ import ConfigService from './config';
 import * as logging from './logging';
 import {renderServer} from './renderServer';
 import {renderStream} from './renderStream';
+import {PollingConfig} from './types';
 
 dotenv.config();
 Sentry.init({dsn: process.env.SENTRY_DSN});
 
+const defaultPollingConfig: PollingConfig = {
+  /**
+   * Poll ever 5 seconds on boot
+   */
+  bootInterval: 5 * 1000,
```

Comment:
Would add the unit of this interval at the end as a comment:

```js
bootInterval: 5 * 1000 /*ms*/
```

Alternative:
```js
const SECONDS = 1000; /*ms*/

...

bootInterval: 5 * SECONDS;
```